set_language.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo 'export LANGUAGE=en_US.UTF-8' >> ~/.bashrc
```

Comment:
Right after launching ubuntu instances by vagrant, or AWS(EC2), the locale isn't set in the instances. For this reason, I've experienced failure installing pyenv versions(such as a command: pyenv install 3.5.2). 
I found the failure was of the unset of locale. and I solved the problem after set the locale.